mods/battle-royale/mod_info.lua

```diff
@@ -1,11 +1,11 @@
 name = "Battle Royale"
-version = 6
-uid = "battle-royale-06"
+version = 7
+uid = "battle-royale-07"
```

Comment:
i think, uid doesn't changes

Reply:
maybe change it to the generated uuid?

Reply:
Game version is used by the user to inform them the version of the mod. The UID is used by the game in case a dependency is required and by the client (Downlords client) to help people auto-download the mod. It can be anything - I use a UID that is not generated as it is easier to read / understand.

Reply:
(it should be updated - otherwise the client doesn't recognize the newer version)